traffic_ops/testing/api/v4/profiles_test.go

```diff
@@ -518,11 +518,31 @@ func GetTestProfiles(t *testing.T) {
 		profileID := resp.Response[0].ID
 
 		// TODO: figure out what the 'Parameter' field of a Profile is and how
```

Comment:
Any reason to keep that first line in there still?

Reply:
I removed this line.